crates/speedy-transform/src/types.rs

```diff
@@ -1,28 +1,42 @@
-use serde::{Deserialize, Serialize};
+use napi::JsFunction;
+use serde::Serialize;
 
-#[derive(Debug, Serialize, Deserialize)]
+#[napi(object)]
+#[derive(Debug)]
+pub struct TransformOutput {
+  pub code: String,
+  pub map: Option<String>,
+}
+
+#[napi(object)]
+#[derive(Serialize)]
 pub struct TransformConfig {
   pub react_runtime: Option<bool>,
   pub babel_import: Option<Vec<BabelImportConfig>>,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[napi(object)]
+#[derive(Serialize)]
 pub struct BabelImportConfig {
   pub from_source: String,
-  pub replace_css: Option<ReplaceCssConfig>,
-  pub replace_js: Option<ReplaceSpecConfig>,
+  pub replace_css: Option<RepalceCssConfig>,
+  pub replace_js: Option<RepalceSpecConfig>,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
-pub struct ReplaceSpecConfig {
-  pub replace_expr: String,
+#[napi(object)]
+#[derive(Serialize)]
+pub struct RepalceSpecConfig {
+  #[serde(skip_serializing)]
+  pub replace_expr: JsFunction,
```

Comment:
Is this a required field? This may cause a performance regression as it was called on each import statement?